capture/parse.py

```diff
@@ -42,7 +42,7 @@ def count_record(file):
             return(tot_records)
 
 
-def parse_fq(output, file, type_f, num_sub, number_records):
+def parse_fq(output, file, type_f, num_sub, number_records, handle):
```

Comment:
This breaks the unit testing. Please add the correct handle argument to the test suite